recursion/Basic Array Questions/Max_arr.py

```diff
@@ -0,0 +1,16 @@
+# Program to find maximum element in an array using recursion
+
+def max_arr_rec(a,n):
+    if n==1: # Base Case if array containing only one element then return it
+        return a[0]
+    else:
+        previous = max_arr_rec(a,n-1)
```

Comment:
please add comments for better understanding. follow as same in other code snippet